### PR TITLE
Fix MySQL, SQL

### PR DESCRIPTION
**Pull request by thebigsmileXD**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
This pull requests fixes - MySQL:
- [x] Not saving
- [x] Not updating 
- [x] Fixed syntax mistake in free plot query
- [x] PHPStorm automatic cleanup
- [x] Saves levels and protects/unloads them if the MySQL server can not be reached.

This pull requests fixes - SQL
- [x] Fixed syntax mistake in free plot query
- [x] PHPStorm automatic cleanup

## Optional information
Has the latest commits in it and is up to date with jasonwynn10/MyPlot.

### CODE REVIEW

**Comment by thebigsmileXD** [comment]
Added a better "reconnect". Got players that told me, when #84 (Server went away) occurs, they can build on any plot. This saves the levels, if the database is fully unreachable, and unloads them. (and currently stops the server, players could still cheat into the level via /home or mw tp or some other things,)

**Comment by jasonwynn10** [comment]
Looks good for the most part, but I'll have to adjust the formatting changes

**Comment by thebigsmileXD** [comment]
I was running PHPStorm cleanup with PocketMine code style

**Comment by jasonwynn10** [comment]
I'll make the changes later to MyPlot's style

File: src/MyPlot/provider/MySQLProvider.php [hunk 1]
```diff
@@ -4,39 +4,55 @@
...
+		ini_set('mysql.connect_timeout', 300);
+		ini_set('default_socket_timeout', 300);
 		$this->plugin = $plugin;
 		parent::__construct($plugin, $cacheSize);
+		$this->settings = $settings;
 		$this->db = new \mysqli($settings['Host'], $settings['Username'], $settings['Password'], $settings['DatabaseName'], $settings['Port']);
 		$this->db->query(
 			"CREATE TABLE IF NOT EXISTS plots (id INT PRIMARY KEY AUTO_INCREMENT, level TEXT, X INT, Z INT, name TEXT, owner TEXT, helpers TEXT, denied TEXT, biome TEXT);");
 		$this->sqlGetPlot = $this->db->prepare("SELECT id, name, owner, helpers, denied, biome FROM plots WHERE level = ? AND X = ? AND Z = ?;");
+		$this->sqlGetIdFromLevelXZ = $this->db->prepare(
```
Comment (jasonwynn10) [inline src/MyPlot/provider/MySQLProvider.php]: Where is this used?

File: src/MyPlot/provider/MySQLProvider.php [hunk 2]
```diff
@@ -210,4 +241,23 @@ public function getNextFreePlot(string $levelName, int $limitXZ = 0) {
...
+			$this->db->close();
+			$this->db->connect($this->settings['Host'], $this->settings['Username'], $this->settings['Password'], $this->settings['DatabaseName'], $this->settings['Port']);
+			if($this->db->ping()){
+				$this->plugin->getLogger()->notice("The MySQL connection has been reestablished!");
+			}else{
+				$this->plugin->getLogger()->critical("The MySQL connection COULD NOT been reestablished!");
+				$this->plugin->getLogger()->critical("Will shut down for safety!");
+				foreach ($this->plugin->getPlotLevels() as $levelname => $values){
+					$this->plugin->getServer()->getLevelByName($levelname)->save(true);
+					$this->plugin->getServer()->unloadLevel($levelname);
```
Comment (jasonwynn10) [inline src/MyPlot/provider/MySQLProvider.php]: parameter calls for Level object, not string

File: src/MyPlot/provider/MySQLProvider.php (hunk 1 above)
Comment (thebigsmileXD) [inline src/MyPlot/provider/MySQLProvider.php]: Nowhere.. could get removed

**Pull request closed without merging**